src/zip.ts

```diff
@@ -167,6 +192,17 @@ export const zipFunction = async function (
     srcPath,
     stat: stats,
   })
+  const durationNs = endTimer(startIntervalTime)
+  const durationMs = roundTimerToMillisecs(durationNs)
+  const logObject = {
+    name,
+    config,
+    featureFlags: functionFlags,
+    // e.g 45m 48s 373ms
+    duration: prettyMilliseconds(durationMs, { formatSubMilliseconds: true }),
```

Comment:
I'm curious about your reasoning for using a human-readable value here. Since system logs are for our own usage only, it feels like a machine-friendly value (like the value in milliseconds or nanoseconds) would be more useful, because we could more easily extract that into a variable in Humio using regex and manipulate it in different ways.

Reply:
I got [this comment](https://github.com/netlify/pod-compute/issues/246#issue-1420815403) mixed up and read it as though we still needed it human friendly. I've changed it to use durationMs.